src/js/utils/stylesheet.js

```diff
@@ -0,0 +1,16 @@
+import document from 'global/document';
+
+export let getStyleElement = function(className) {
```

Comment:
Yeah, since each player needs its own style element I don't think we need a getter. Which reminds me, should we have something that cleans up the style el when the player is disposed? Previously I think it was just relying on the style el being removed with the player div.